docker/puppetserver-standalone/docker-entrypoint.sh

```diff
@@ -10,4 +10,12 @@ for f in /docker-entrypoint.d/*.sh; do
     "$f"
 done
 
+if [ -d /docker-custom-entrypoint.d/ ]; then
+    find /docker-custom-entrypoint.d/ -type f -name "*.sh" \
+        -exec chmod +x {} \;
```

Comment:
Also tried just `chmod +x /docker-custom-entrypoint.d/*` but that too fails when the directory is empty.